syftlib/src/main/java/org/openmined/syft/networking/clients/HttpClient.kt

```diff
@@ -8,6 +8,12 @@ import org.openmined.syft.networking.requests.NetworkingProtocol
 import retrofit2.Retrofit
 import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
 
+/**
+ * Creates http api client for federated related endpoints.
```

Comment:
"Creates a retrofit api client for connecting to PyGrid endpoints" would be better